demos/python_demos/object_detection_demo_centernet/README.md

```diff
@@ -0,0 +1,54 @@
+# Object Detection CenterNet Demo
+
+This demo showcases Object Detection with CenterNet. The task is to identify objects as axis-aligned boxes in an image.
+CenterNet model an object as a single point - the center point of its bounding box and uses keypoint estimation to find center points and regresses to object size. The same approach is used to estimate 3D bounding box and pose.
+
+## How It Works
+
+On the start-up, the application reads command-line parameters and loads a network to the Inference
+Engine. Upon getting a frame from the OpenCV VideoCapture, it performs inference and displays the results.
+
+## Running
+
+Running the application with the <code>-h</code> option yields the following usage message:
+
+```
+usage: object_detection_demo_centernet.py [-h] -m MODEL [-i INPUT [INPUT ...]]
+                                          [-l CPU_EXTENSION] [-d DEVICE]
+                                          [--labels LABELS]
+                                          [-pt PROB_THRESHOLD] [--no_show]
+
+Options:
+  -h, --help            Show this help message and exit.
+  -m MODEL, --model MODEL
+                        Required. Path to an .xml file with a trained model.
+  -i INPUT [INPUT ...], --input INPUT [INPUT ...]
+                        path to video or image/images
+  -l CPU_EXTENSION, --cpu_extension CPU_EXTENSION
```

Comment:
-l option is not needed for OpenVINO R4 as it does not require cpu_extension library